src/scenario.h

```diff
@@ -302,7 +302,7 @@ typedef struct {
 	char custom_strings[0x8000];
 	uint32 game_ticks_1;
 	rct_ride rides[255];
-	uint16 word_01388698;
+	uint16 word_01388698; // TODO: rename to match RCT2_ADDRESS_SAVED_AGE ???
```

Comment:
Something weird is going on with `RCT2_ADDRESS_SAVED_AGE `. In the title sequence, it attempts to get a ride from the address (ride index is 255). Also, I noticed `RCT2_ADDRESS_SAVED_VIEW_X` is sometimes modified as an `uint16`, and sometimes as a `sint16`.